### ISSUE TITLE
not possible to distinguish between not found and wrong name error

### ISSUE THREAD

**Issue by jurajsimbs** [fogine/couchbase-odm#8]
in modelManager.js:
```js
    if (type == 'string') {
        if (!this.models.hasOwnProperty(name)) {
            throw new Error("Model `" + name + "` not found.");
        }
        return this.models[name];
    } else {
        throw new Error("Expected string name of the Model");
    }
```

for example I'd like catch notFound error and in that case load model from somewhere. Wrong name can stay general `Error` and I'd pass that to main loop 

what do you think?

**Comment by fogine** [comment]
I agree, the method should definitely throw specific error objects.

Thank you for the suggestion.

**Comment by fogine** [comment]
Is available on both the `develop` and `v1.0` branches.

if model is not found, it throws `CouchbaseODM.errors.ModelNotFoundError` for any other error it throws `CouchbaseODM.errors.ModelManagerError`.
